src/test/java/nl/tudelft/contextproject/camera/LiveCameraConnectionTest.java

```diff
@@ -97,74 +101,104 @@ public void testGetCurrentCameraSettings() {
      * @throws InterruptedException Due to {@link Thread#sleep(long)}
      */
     @Test
-    public void testAbsPanTilt() throws InterruptedException {
-        if (doTests) {
-            assertTrue(connection.absPanTilt(29965, 28965));
+    public void testAbsPanTilt() throws Exception {
+        if (!testLive) {
+            URL absPanTiltURL = connection.buildPanTiltHeadControlURL("%23APS750D71251D2");
+            URL curPanTiltURL = connection.buildPanTiltHeadControlURL("%23APC");
+            doReturn("aPS750D71251D2").when(connection, "sendRequest", absPanTiltURL);
+            doReturn("aPC750D7125").when(connection, "sendRequest", curPanTiltURL);
+        }
+        assertTrue(connection.absPanTilt(29965, 28965));
+        if (testLive) {
             Thread.sleep(4000);
-            int[] curSet = connection.getCurrentPanTilt();
-            assertWithinMaxOffset(29965, curSet[0]);
-            assertWithinMaxOffset(28965, curSet[1]);
         }
+        int[] curSet = connection.getCurrentPanTilt();
+        assertWithinMaxOffset(29965, curSet[0]);
+        assertWithinMaxOffset(28965, curSet[1]);
     }
 
     /**
      * Tests {@link LiveCameraConnection#absPan(int)}.
      * @throws InterruptedException Due to {@link Thread#sleep(long)}
      */
     @Test
-    public void testAbsPan() throws InterruptedException {
-        if (doTests) {
-            int[] before = connection.getCurrentPanTilt();
-            assertTrue(connection.absPan(31965));
+    public void testAbsPan() throws Exception {
+        int[] before = connection.getCurrentPanTilt();
+        if (!testLive) {
+            URL absPanTiltURL = connection.buildPanTiltHeadControlURL("%23APS7CDD80001D2");
+            URL getPanTiltURL = connection.buildPanTiltHeadControlURL("%23APC");
+            doReturn("aPS").when(connection, "sendRequest", absPanTiltURL);
+            doReturn("aPC7CDD8000").when(connection, "sendRequest", getPanTiltURL);
+        }
+        assertTrue(connection.absPan(31965));
+        if (testLive) {
             Thread.sleep(4000);
-            int[] after = connection.getCurrentPanTilt();
-            assertWithinMaxOffset(before[1], after[1]);
-            assertWithinMaxOffset(31965, after[0]);
         }
+        int[] after = connection.getCurrentPanTilt();
+        assertWithinMaxOffset(before[1], after[1]);
+        assertWithinMaxOffset(31965, after[0]);
     }
 
     /**
      * Tests {@link LiveCameraConnection#absTilt(int)}.
      * @throws InterruptedException Due to {@link Thread#sleep(long)}
      */
     @Test
-    public void testAbsTilt() throws InterruptedException {
-        if (doTests) {
-            int[] before = connection.getCurrentPanTilt();
-            assertTrue(connection.absTilt(31965));
+    public void testAbsTilt() throws Exception {
+        int[] before = connection.getCurrentPanTilt();
+        if (!testLive) {
+            URL absPanTiltURL = connection.buildPanTiltHeadControlURL("%23APS80007CDD1D2");
```

Comment:
No. In the case you hadn't noticed: the values used in the command that makes up the URL are different every time.